connector-pagerduty/src/main/java/com/redhat/cloud/notifications/connector/pagerduty/PagerDutyCloudEventDataExtractor.java

```diff
@@ -37,16 +31,11 @@ public void extract(Exchange exchange, JsonObject cloudEventData) throws Illegal
             exchange.setProperty(TARGET_URL, providedUrl);
         }
 
+        UrlValidationHelper.validateTargetUrl(exchange);
```

Comment:
We did allow `http` before with PagerDuty and now we're forbidding it. Did you check in the DB that no one is using an `http` URL?

Reply:
I think the only requirement was that the test cases needed to use non-TLS URLs (they gave a cert error otherwise) but maybe that's not the case anymore